eox_core/api/v1/tests/integration/test_views.py

```diff
@@ -0,0 +1,404 @@
+"""
+Integration test suite for the API v1 views.
+"""
+
+from __future__ import annotations
+
+import requests
+from ddt import data as ddt_data
+from ddt import ddt, unpack
+from django.http import HttpResponse
+from django.urls import reverse
+from faker import Faker
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+fake = Faker()
+
+
+API_TIMEOUT = 5
+CLIENT_ID = "client_id"
+CLIENT_SECRET = "client_secret"
+FAKE_USERS_DATA = iter(
+    [
+        {
+            "username": fake.unique.user_name(),
+            "email": fake.unique.email(),
+            "fullname": fake.unique.name(),
+            "password": fake.unique.password(),
+            "activate_user": True,
+        }
+        for _ in range(100)
+    ]
+)
+
+
+class BaseAPIIntegrationTest(APITestCase):
```

Comment:
@BryanttV since we are not using the API client to test the APIs, I think we should just use the TestCase class instead

Reply:
Done: https://github.com/eduNEXT/eox-core/commit/f82e677570ec7bb9bb7b4fbb1f28a090d43db807